bootstrap/utils/utils.go

```diff
@@ -161,3 +167,47 @@ func DeepCopy(src any, dest any) error {
 	}
 	return nil
 }
+
+func LoadFile(path string, contents *any) error {
+	var fileBytes []byte
+	var err error
+
+	lowerPath := strings.ToLower(path)
+
+	if strings.Index(lowerPath, "http") == 0 {
```

Comment:
I was thinking of a better way to ensure that the path is a URL and not a file name like `http-stuff.yaml` would be to use url.Parse. This now gives you the parsed URL to log the redacted url. 
```suggestion
	parsedUrl, err := urlParse(lowerPath, "http"
	if err == nil {
```